assets/css/desk-about.css

```diff
@@ -0,0 +1,202 @@
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  font-family: "Lato", sans-serif;
+}
+
+.summit-page {
+  background-color: #f4efef;
+  width: 100%;
+}
+
+.summit-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 0;
+}
+
+.presents {
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-bottom: 5rem;
+}
+
+.conference-intro {
+  text-align: center;
+  padding-bottom: 7rem;
+}
+
+.first-text {
+  text-align: center;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 28px;
+  color: #272a31;
+  background-color: #fff;
+  padding: 12px 8px 26px;
+  width: 70%;
+  border-radius: 8px;
+  margin-bottom: 2rem;
+}
+
+.second-text {
+  text-align: center;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 28px;
+  color: #272a31;
+  width: 50%;
+  padding-bottom: 10px;
+}
+
+.mail {
+  color: #272a31;
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.logo-page {
+  background-color: #fff;
+  padding-bottom: 2rem;
+  border-bottom: 1px solid #272a31;
+}
+
+.logo-page-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 3rem;
+}
+
+.page-intro {
+  color: #272a31;
+  font-size: 35px;
+  font-weight: 600;
+  line-height: 30px;
+  padding-bottom: 10px;
+  margin-bottom: 3rem;
+}
+
+.page-intro::after {
+  content: '';
+  height: 2px;
+  width: 50px;
+  background-color: #ec5242;
+  bottom: 0;
+  border-radius: 10px;
+  margin: 0 auto;
+  display: block;
+  margin-top: 0.8em;
+}
+
+.logo-text {
+  text-align: center;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 28px;
+  color: #272a31;
+  width: 60%;
+}
+
+.logo-display {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #272a31;
+  margin-top: 6rem;
+  border-radius: 6px;
+}
```

Comment:
- [ ] Please ensure that border weight matches the given design requirement :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student about lines](https://github.com/Kwamay/Capstone-Project/assets/43528594/3c75c8dc-0ca5-4646-b475-59b67ffbf6bf)

</td>
      <td>

![Design About lines](https://github.com/Kwamay/Capstone-Project/assets/43528594/f3654a7e-f508-48b0-a873-b114041135e4)

</td>
    </tr>
  </table>